apps/dialtone-documentation/docs/.vuepress/baseComponents/AllTokens.vue

```diff
@@ -173,8 +173,8 @@ const filterTokens = () => {
         const searchRegex = searchValue.replace(/\(/, '\\(').replace(/\)/, '\\)'); // escape parenthesis
         const regex = new RegExp(searchRegex, 'i');
         if (regex.test(category)) return true;
-        const { name, tokenValue, description } = token;
```

Comment:
Previously it filtered by description, not sure if it was intended, but now it will not.

Reply:
I think it was intended right nina?

Reply:
yes!

Reply:
Yes! It was intended. Name, value, and keywords are the only ways to search for a Token.